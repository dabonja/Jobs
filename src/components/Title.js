import React from 'react';

const Title = ({title,title2}) => {
    return (
        <div className="jumbotron jumbotron-fluid">
        <div className="container">
          <h1 className="display-4">{title}</h1>
          <p className="lead">{title2}</p>
        </div>
      </div>
    )
}
export default Title