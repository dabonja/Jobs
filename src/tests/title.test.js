import Title from '../components/Title';
import renderer from 'react-test-renderer';

it('renders correctly', ()=>{
    const tree = renderer.create(<Title title="Dabenda"/>)
    .toJSON();
    expect(tree).toMatchSnapshot()
})