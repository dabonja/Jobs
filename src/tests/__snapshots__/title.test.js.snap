// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Dabenda
  </h1>
</div>
`;
